src/frontend/src/flows/dappsExplorer/teaser.ts

```diff
@@ -70,13 +70,18 @@ const marqueeList = (dapps: KnownDapp[]): TemplateResult => {
   >
     ${rows.map((rowDapps, i) => {
       const rowContent = rowDapps.map(
-        // XXX: it's important not to lazy load the image, otherwise
-        // images start appearing before they're loaded. This then
-        // shows an empty space where the image suddenly pops seconds
-        // later.
         ({ logoSrc, name }) => html`<div class="c-marquee__item">
           <div class="c-marquee__image-container">
-            <img src=${logoSrc} alt="${name}" class="c-marquee__image" />
+            <img
+              src=${logoSrc}
+              alt="${name}"
+              class="c-marquee__image c-marquee__image--loading"
+              @load=${(e: Event) => {
```

Comment:
Personally, I would have preferred this change in another PR.

Let's discuss it with the team and merge this as is.

Reply:
Without this it looks very ugly, but we can separate it out if that helps.

Reply:
Let's discuss it this afternoon. There is no reason to separate it now.